packages/admin/cms-admin/src/generator/future/generateFormField.ts

```diff
@@ -159,6 +161,13 @@ export function generateFormField(
             ${validateCode}
             {(props) =>
                 <FinalFormSelect ${config.readOnly ? readOnlyPropsWithLock : ""} {...props}>
+                ${
+                    !required
+                        ? `<MenuItem value="" >
+                            <FormattedMessage id="${instanceGqlType}.${name}.pleaseSelect" defaultMessage="Please select" />
```

Comment:
We should have a generic message for this in https://github.com/vivid-planet/comet/blob/next/packages/admin/admin/src/messages.ts